### PR TITLE
Add Trove classifiers.

### PR DESCRIPTION
**Pull request by kdeldycke**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,6 +32,29 @@
...
+        # https://en.wikipedia.org/wiki/CPython#Version_history
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```
Comment (jhermann) [inline setup.py]: The versions should be in sync with the Travis config (i.e. change one of them).

Comment (kdeldycke) [inline setup.py]: Ah yes, you're right. Just sync version with Travis jobs in b6267e2dfa731ed1fc6c9cdd6aff0564a4aa386a .

**Comment by kdeldycke** [comment]
This PR is ready to be merged.

**Comment by kdeldycke** [comment]
Thanks @samgiles for the merge! :)

**Pull request merged**